qpc/cred/add.py

```diff
@@ -128,4 +119,4 @@ def _build_data(self):
         self.req_payload = build_credential_payload(self.args, self.args.type)
 
     def _handle_response_success(self):
-        print(_(messages.CRED_ADDED % self.args.name))
+        log.info(_(messages.CRED_ADDED % self.args.name))
```

Comment:
A couple "best practices" suggestions here! 😅 

First, if we're using gettext translations (the `_` function), then we should never pass variables like `self.args.name` into it because that would result in a whole new string to translate _including_ the value of that variable, for every variable it might encounter. Instead, we should always translate only "tokenized" strings with variable placeholders. You then format the variables into the string _after_ translation has occurred (after the `_` function executes).

For example, if I want `"My name is %s" % name` to be translated correctly, I would use `_("My name is %s") % name`.

If you have a string that takes _multiple_ variables, it's also important also to _name_ the tokens. Not all language grammars use the same word order, and using tokens enables the translator to rearrange the content correctly for that language. Think about how some languages typically use adjective-noun but other languages typically use noun-adjective.

For example, `_("The %(color)s %(food)s is %(quality)s.") % {"color": "red", "food": "apple", "quality": "delicious"}`.

Second, instead of using `%` to format the string, we should pass the tokenized string and its variables as separate arguments to the `logger.info` call. Why? Well, if you use `%` that means the string is always _immediately_ formatted _before_ sending it to the logger's function. However, you might have a logging configuration that isn't going to output a particular call. Consider, for example, that we often don't _actually_ output `logger.debug` calls. If you simply pass the arguments to the logger function, it will only format the string _if necessary_. If the logging config would skip that call (like `debug`), then it also skips formatting, which saves memory and CPU time. It doesn't save a _lot_, especially for a small message like this, but it adds up, and it's a good habit to follow in general.

Extending the previous example, that would look like:

```python
logger.info(
    _("The %(color)s %(food)s is %(quality)s."),
    {"color": "red", "food": "apple", "quality": "delicious"}
)
```

(Whitespace added to make the separation obvious.)

```suggestion
        logger.info(_(messages.CRED_ADDED), self.args.name)
```